scripts/pr-shepherd.sh

```
@@ -0,0 +1,98 @@
+# Usage: ./pr-shepherd.sh
+# Output: JSON report of PR statuses and needed actions
+#
+# Designed to run as a cron job every 30 minutes.
+# Checks CI status, review status, and age for all open PRs
+# authored by kaylee-mistystep (sprite account).
+
+set -euo pipefail
+
+ORG="misty-step"
+AUTHOR="kaylee-mistystep"
+STATE_FILE="/tmp/pr-shepherd-state.json"
```

Shellcheck (0.11.0)

[warning] STATE_FILE appears unused. Verify use (or export if used externally).

(SC2034)